### DISCUSSION TITLE
add -g, --github <user> flag

### DISCUSSION THREAD

**Discussion by casperdcl** [FiloSottile/age#283] [UX reports]
Similar to https://github.com/S2-/sshenc.sh#encrypt-a-file-using-the-public-key-of-a-github-user

would simplify:

```bash
# old
curl -sL https://github.com/someusername.keys | age -R - -o output.file input.file
# new
age -g someusername -o output.file input.file
```

**Comment by str4d** [comment]
This was present in an early draft of the age CLI design, and I had implemented it as an unstable feature in `rage`. However, we decided over the new year to drop the "encrypt to GitHub users" feature. It relies on support for encrypting to recipients at HTTPS URLs, and there are **many** decisions that need to be made when downloading a file (e.g. what roots to trust?) that go beyond the APIs we want to focus on here, and are better suited to dedicated URL-fetching tools like `curl` or `wget`.

As you note, encrypting to a GitHub user in a single command can still be done with pipes. It can also be achieved with [process substitution](https://en.wikipedia.org/wiki/Process_substitution):

```
cat input.file | age -R <(curl -sL https://github.com/someusername.keys) -o output.file

**Comment by casperdcl** [comment]
er, why not launch a `curl`/`wget` process from `age` (throwing an error if the binaries are not found)?
